docs/content/guide/interpolation.ngdoc

```diff
@@ -26,6 +26,15 @@ normal {@link ng.$rootScope.Scope#$digest digest} cycle.
 
 Note that the interpolateDirective has a priority of 100 and sets up the watch in the preLink function.
 
+### How the string representation is computed
+
+If something other than a String is interpolated, the string is computed as follows:
+- If the object is not a `Number`, `Date` or `Array`, $interpolate looks for a custom `toString()`
+function on the object, and uses that. Custom means that
+`myObject.toString() !== `Object.prototype.toString()`.
+
+- If the above doesn't apply, then `JSON.stringify` is used.
```

Comment:
There is no mention about `null`/`undefined` being converted to the empty string.